la_palma_overview/la_palma_overview_video.py

```diff
@@ -120,8 +120,8 @@ def date_path(date, base='', subdir=''):
     return os.path.join(
         base,
         str(date.year),
```

Comment:
maybe add `'{:04d}'.format(date.year)` for analogy and completeness in case somebody wants to call this for 2000 years in the past?